@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -691,12 +691,32 @@ export interface MarkerTechniqueParams extends BaseTechniqueParams {
     wrappingMode?: DynamicProperty<"None" | "Character" | "Word">;
     /**
      * Text position regarding the baseline.
+     *
+     * @deprecated Use [[hPlacement]] attribute, which defines label text placement in exactly
+     * opposite way, text that were `Left` aligned are by deduction `Right` placed relative to
+     * the label icon.
      */
     hAlignment?: DynamicProperty<"Left" | "Center" | "Right">;
     /**
      * Text position inside a line.
+     *
+     * @deprecated Use [[vPlacement]] attribute instead.
      */
     vAlignment?: DynamicProperty<"Above" | "Center" | "Below">;
+    /**
+     * Text position regarding the baseline.
+     *
+     * @deprecated Use [[hPlacement]] attribute, which defines label text placement in exactly
+     * opposite way, text that were `Left` aligned are by deduction `Right` placed relative to
+     * the label icon.
+     */
+    hPlacement?: DynamicProperty<"Left" | "Center" | "Right">;
+    /**
+     * Text position inside a line.
+     *
+     * @deprecated Use [[vPlacement]] attribute instead.
```

Comment:
As we've discussed by webex, one step further to make text styling the same as in native is to unify horizontal and vertical placement into one attribute.

Reply:
Totally agree. Also see the discussion above about supporting arrays of placements in the future.

Reply:
See my comments in original ticket -> probably there is a handy solution how the placement should be defined, but... First of all we do not have (AT ALL) support for storing alignment/placement priorities, we do not have support for solving such labels allocation in screen space, we will perform much more computation to find good allocation spot, and last but not least it is not the scope of this ticket! Please read original description.